src/io/opendss.jl

```diff
@@ -198,8 +208,8 @@ function dss2tppm_load!(tppm_data::Dict, dss_data::Dict, import_all::Bool)
             loadDict["qd"] = PMs.MultiConductorVector(parse_array(defaults["kvar"] / 1e3, nodes, nconductors))
             loadDict["status"] = convert(Int, defaults["enabled"])
 
-            active_phases = join(["$n" for n in 1:nconductors if nodes[n] > 0], ".")
-            loadDict["source_id"] = "load.$(defaults["name"]).$(active_phases)"
+            loadDict["active_phases"] = [nodes[n] > 0 ? 1 : 0 for n in 1:nconductors]
```

Comment:
Here I would recommend a sparse vector of phase ints rather than booleans, i.e. `[2,3]` instead of `[0,1,1]`.

Then we can use it interchangeably with the sets semantics proposed above.